pipelines/gke/scripts/longhorn-setup.sh

```
@@ -26,6 +26,32 @@ set_kubeconfig_envvar(){
     gcloud container clusters get-credentials `terraform -chdir=${TF_VAR_tf_workspace}/terraform output -raw cluster_name` --zone `terraform -chdir=${TF_VAR_tf_workspace}/terraform output -raw cluster_zone` --project ${TF_VAR_gcp_project}
 }
 
+print_out_cluster_info(){
+  gcloud container clusters describe `terraform -chdir=${TF_VAR_tf_workspace}/terraform output -raw cluster_name` --zone `terraform -chdir=${TF_VAR_tf_workspace}/terraform output -raw cluster_zone` --format="value(currentNodeVersion)"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)